doc/install/ceph-deploy/quick-start-preflight.rst

```diff
@@ -302,7 +302,7 @@ Once you have finished configuring ``iptables``, ensure that you make the
 changes persistent on each node so that they will be in effect when your nodes
 reboot. For example::
 
-	/sbin/service iptables save
+	/sbin/service iptables-save
```

Comment:
see https://docs.fedoraproject.org/en-US/Fedora/11/html/Security_Guide/sect-Security_Guide-IPTables-Saving_IPTables_Rules.html and i don't think there is a service named "iptables-save".